lib/tasks/check_changelog.rake

```diff
@@ -34,18 +65,15 @@ namespace "check" do
       # has been changed
       if version_changed?( "#{osc_checkout_dir}/#{package_name}.spec" )
         Dir.chdir(osc_checkout_dir) do
-          # Tags described in https://github.com/openSUSE/osc-plugin-factory/blob/e12bc02e9817277335ce6adaa8e8d334d03fcc5d/check_tags_in_requests.py#L63
           cmd = "osc -A '#{obs_api}' cat " \
             " '#{obs_sr_project}' '#{package_name}' '#{package_name}.changes' "\
             "| diff - '#{package_name}.changes'"
           puts cmd if verbose
           ret = `bash -c '#{cmd}'`
-          unless ret.match(/(bnc|fate|boo|bsc|bgo)#[0-9]+/i) ||
-                 ret.match(/cve-[0-9]{4}-[0-9]+/i)
+          unless ID_MATCHERS.any?{|m| ret.match(m) }
```

Comment:
I use exactly same regexp as build service use.
So I use same matching as build service and what pass build service should pass also here 
https://github.com/openSUSE/open-build-service/blob/ca026f81b20709156ead8e3859fab66259bac38a/src/api/app/models/issue_tracker.rb#L63